src/components/article/ListPagination.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Pagination, Grid } from 'semantic-ui-react';
+import '../../styles/Articles.scss';
+
+
+class ListPagination extends Component {
+    static currentLocalStorage = (localStorage) => {
+        const initialPage = 1;
+        let currentPage = localStorage.getItem('page');
+        if (currentPage !== null) {
+            return currentPage;
+        } else {
+            return currentPage = initialPage;
+        }
+    }
+    onPageChange = (event, data) => {
+        const { fetchArticles } = this.props;
+        const url = 'https://ah-backend-summer-staging.herokuapp.com/api/v1/articles';
+        const initialIndex = 0;
+        const limit = (count, index) => 
+            `limit=${count}&offset=${index ? index * count : initialIndex}`;
+        const page = data.activePage;
+        const maxPageCount = 10;
+        const startPage = 1;
+        const pagination = `?${limit(maxPageCount, page - startPage)}`;
+        fetchArticles(url + pagination);
+        localStorage.setItem('page', page);
+    };
+    render() {
+        const { articleCount } = this.props;
+        const maxPageCount = 10;
```

Comment:
repetition. `const maxPageCount = 10;` line `24` and `32`.

Reply:
worked on, thanks